kusto/ingest/internal/resources/resources.go

```diff
@@ -350,17 +359,24 @@ func (m *Manager) fetchRetry(ctx context.Context) {
 		cancel()
 		if err != nil {
 			attempts++
-			//log.Printf("problem fetching the resources from Kusto Mgmt(attempt %d): %s", attempts, err)
 			time.Sleep(10 * time.Second)
```

Comment:
You sleep only to loop back into the exist condition, i.e. worst case scenario you'd sleep 4 times rather then 3

Reply:
Fixed

Reply:
Not really ;)

Reply:
Oops! fixed